lighthouse-core/test/results/sample_v2.json

```diff
@@ -191,6 +191,15 @@
       "numericValue": 16,
       "displayValue": "20 ms"
     },
+    "cumulative-long-queuing-delay": {
+      "id": "cumulative-long-queuing-delay",
+      "title": "Cumulative Long Queuing Delay",
+      "description": "[Experimental metric]. Sum of Task Lengths beyond 50ms, between First Contentful Paint and Time To Interactive.",
+      "score": 1,
+      "scoreDisplayMode": "numeric",
+      "numericValue": 116.79800000000023,
```

Comment:
This long floating point values look very strange. Am I supposed to limit the digits on this somehow?

Reply:
> This long floating point values look very strange. Am I supposed to limit the digits on this somehow?

It doesn't necessarily hurt anything (if/when we include it in the HTML report it'll get formatted nicely), but for most of these, yeah, it's manually rounded to a point that represents the actually available (and/or actually meaningful) precision (e.g. [nearest 10ms](https://github.com/GoogleChrome/lighthouse/blob/4a74d1731166985affb35ecdcea9ccf54f3fea19/lighthouse-core/audits/byte-efficiency/byte-efficiency-audit.js#L184), 1 ms, etc)